test/fixtures/grains/results.json

```diff
@@ -4,7 +4,7 @@
     "tests": [
         {
             "name": "beginning squares",
-            "test_code": "@test on_square(1) == 1\n@test on_square(2) == 2\n@test on_square(3) == 4\n@test on_square(4) == 8\n@test on_square(16) == 32768\n@test on_square(32) == 2147483648\n@test total_after(1) == 1\n@test total_after(3) == on_square(1) + on_square(2) + on_square(3)",
+            "test_code": "@test on_square(1) == 1\n@test on_square(2) == 2\n@test on_square(3) == 4\n@test on_square(4) == 8\n@test on_square(16) == 32768\n...",
```

Comment:
@exercism/reviewers Can someone approve and merge this, please?

The PR truncates the emitted test_code from test sets that contain many passing tests. The PR is part of an ongoing effort to let the exercism web editor support some of our parametrized tests. You can see the effect of the PR in the diff of the fixture/snapshot test above.

The PR also fixes a function that generates the test_code strings from a given test and ensures that that codepath is tested in CI.

Contrary to normal exercism practice, this PR should not be squashed (though it doesn't matter all that much).